docs/how-to-guides/isis-developer-guides/contributing-to-isis.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to ISIS
+This document serves as a concise guide to contribute to ISIS.
```

Comment:
Perhaps phrased like `.. guide on how to contribute to ISIS` or `.. guide on contributing to ISIS`?